rDNA/tests/testthat/test-zzz.R

```diff
@@ -0,0 +1,12 @@
+context("cleanup")
+
+test_that("str_length is number of characters", {
+  expect_that(nchar("a"), equals(1))
```

Comment:
I like the full `expect_that` notation. Makes it easier to read. Perhaps add to the other places as well?